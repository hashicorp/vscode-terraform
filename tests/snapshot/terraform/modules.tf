module "servers" {
  source = "./app-cluster"

  servers = 5
}

resource "aws_elb" "example" {
  # ...

  instances = module.servers.instance_ids
}
