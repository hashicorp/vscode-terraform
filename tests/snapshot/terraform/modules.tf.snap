>module "servers" {
#^^^^^^ source.terraform meta.block.terraform entity.name.type.terraform
#      ^ source.terraform meta.block.terraform
#       ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.begin.terraform
#        ^^^^^^^ source.terraform meta.block.terraform entity.name.tag.terraform
#               ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.end.terraform
#                ^ source.terraform meta.block.terraform
#                 ^ source.terraform meta.block.terraform punctuation.section.block.begin.terraform
>  source = "./app-cluster"
#^^ source.terraform meta.block.terraform
#  ^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#        ^ source.terraform meta.block.terraform variable.declaration.terraform
#         ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#          ^ source.terraform meta.block.terraform variable.declaration.terraform
#           ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#            ^^^^^^^^^^^^^ source.terraform meta.block.terraform string.quoted.double.terraform
#                         ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>
>  servers = 5
#^^ source.terraform meta.block.terraform
#  ^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#         ^ source.terraform meta.block.terraform variable.declaration.terraform
#          ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#           ^ source.terraform meta.block.terraform variable.declaration.terraform
#            ^ source.terraform meta.block.terraform constant.numeric.integer.terraform
>}
#^ source.terraform meta.block.terraform punctuation.section.block.end.terraform
>
>resource "aws_elb" "example" {
#^^^^^^^^ source.terraform meta.block.terraform entity.name.type.terraform
#        ^ source.terraform meta.block.terraform
#         ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.begin.terraform
#          ^^^^^^^ source.terraform meta.block.terraform entity.name.tag.terraform
#                 ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.end.terraform
#                  ^ source.terraform meta.block.terraform
#                   ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.begin.terraform
#                    ^^^^^^^ source.terraform meta.block.terraform entity.name.tag.terraform
#                           ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.end.terraform
#                            ^ source.terraform meta.block.terraform
#                             ^ source.terraform meta.block.terraform punctuation.section.block.begin.terraform
>  # ...
#^^ source.terraform meta.block.terraform
#  ^ source.terraform meta.block.terraform comment.line.number-sign.hcl punctuation.definition.comment.terraform
#   ^^^^ source.terraform meta.block.terraform comment.line.number-sign.hcl
>
>  instances = module.servers.instance_ids
#^^ source.terraform meta.block.terraform
#  ^^^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#           ^ source.terraform meta.block.terraform variable.declaration.terraform
#            ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#             ^ source.terraform meta.block.terraform variable.declaration.terraform
#              ^^^^^^ source.terraform meta.block.terraform support.constant.terraform
#                    ^ source.terraform meta.block.terraform keyword.operator.accessor.terraform
#                     ^^^^^^^ source.terraform meta.block.terraform variable.other.member.terraform
#                            ^ source.terraform meta.block.terraform keyword.operator.accessor.terraform
#                             ^^^^^^^^^^^^ source.terraform meta.block.terraform variable.other.member.terraform
>}
#^ source.terraform meta.block.terraform punctuation.section.block.end.terraform
>